orca/src/llm/openai.rs

```diff
@@ -151,20 +181,25 @@ pub struct OpenAI {
     ///
     /// The total length of input tokens and generated tokens is limited by the model's context length. [Example Python code](https://github.com/openai/openai-cookbook/blob/main/examples/How_to_count_tokens_with_tiktoken.ipynb) for counting tokens.
     max_tokens: u16,
+
+    /// The format of the returned data. With the new update, the response can be set to a JSON object.
+    /// https://platform.openai.com/docs/guides/text-generation/json-mode
+    response_format: Option<ResponseFormat>,
```

Comment:
Nitpick:
Hmm, I am wondering if it will be better without Option...
```suggestion
    response_format: ResponseFormat,
```

Maybe it is possible to override the default `zero` value. So users don't have to set it, at the same time may read more about default format.